core/src/test/java/org/elasticsearch/ElasticsearchExceptionTests.java

```diff
@@ -67,7 +67,7 @@ public void testToXContent() throws IOException {
         // in the JSON. Since the stack can be large, it only checks the beginning of the JSON.
         assertExceptionAsJson(e, true, startsWith("{\"type\":\"exception\",\"reason\":\"foo\"," +
                 "\"caused_by\":{\"type\":\"illegal_state_exception\",\"reason\":\"bar\"," +
-                "\"stack_trace\":\"java.lang.IllegalStateException: bar"));
+                "\"stack_trace\":\"java.lang.IllegalStateException: bar\\n\\tat org.elasticsearch."));
```

Comment:
> did we change the output here or did we just improve the test?

A little improvement to the test.

> not sure if that will work cross platform either

Right, I changed that, it should be ok on Windows now.